src/main/java/jenkins/advancedqueue/priority/strategy/PriorityJobProperty.java

```diff
@@ -119,7 +121,15 @@ public ListBoxModel getPriorities() {
 		public boolean isUsed(Job<?,?> owner) {
 			PriorityConfiguration configuration = PriorityConfiguration.get();
 			JobGroup jobGroup = configuration.getJobGroup(dummyCallback, owner);
-			return jobGroup != null;
+			if(jobGroup != null && jobGroup.isUsePriorityStrategies()) {
```

Comment:
I think there's a logic glitch here.
the method returns false when jobGroup != null and disabled usePriorityStrategies. I would expect to get true in such case